Exercise_8/api/Api.js

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Come on. Do not start a file with an empty row. But did ES-Lint complain about it?